split.Console/Models/Blackjack/BlackjackGame.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+
+namespace split.Console.Models.Blackjack
+{
+    public class BlackjackGame : Game
+    {
+        Deck CardDeck;
+        BlackjackPlayer Dealer;
+
+        public BlackjackGame(int decks) : base(GameType.Blackjack)
+        {
+            CardDeck = new Deck(decks);
+            Dealer = new BlackjackPlayer("Dealer", null);
+        }
+
+        /// <summary>
+        /// Adds a newly created player to the game
+        /// </summary>
+        /// <param name="newPlayer">Created player to be added to the game</param>
+        public void AddPlayer(BlackjackPlayer newPlayer)
+        {
+            Players.Add(newPlayer);
+        }
+
+        /// <summary>
+        /// Puts the game into an active state
+        /// </summary>
+        /// <returns>True is game has been started</returns>
+        public new bool StartGame()
+        {
+            CardDeck.Shuffle();
+            return base.StartGame();
+        }
+
+        /// <summary>
+        /// Deals 2 Cards to each player including the dealer
+        /// </summary>
+        public void DealRoundOfCards()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                foreach (BlackjackPlayer player in Players)
+                {
+                    player.AddCardToHand(CardDeck.DealCard());
+                }
+                Dealer.AddCardToHand(CardDeck.DealCard());
+            }
+        }
+
+        public void MakeMove(BlackjackPlayer player, BlackjackMove move, int bet = 0)
+        {
+            switch (move)
```

Comment:
This should also return a Player object since methods that are being called here should also return a Player object, it is just a matter of returning the Player object at the end.